<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom RPC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.png</normaloff>assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 53, 147);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="ButtonLast">
           <property name="text">
            <string>Last</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ClientID">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(91, 110, 225);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>ClientID</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0" colspan="2">
           <widget class="QLineEdit" name="State">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>State</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="LargeImageKey">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Large Image Key</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="ButtonName1">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Button 1 Name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="SmallImageKey">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Small Image Key</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="ButtonUrl2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Button 2 URL</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="Timestamp">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(238, 238, 236);</string>
              </property>
              <property name="text">
               <string>Timestamp</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <widget class="QLineEdit" name="ButtonName2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Button 2 Name</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="ButtonUrl1">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Button 1 URL</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="SmallImageText">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Small Image Text</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="LargeImageText">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Large Image Key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="Details">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(91, 110, 225);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Details</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ButtonClear">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(136, 138, 133);</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonStop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(204, 0, 0);</string>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/discord.png</normaloff>assets/discord.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonStart">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(115, 210, 22);</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/discord.png</normaloff>assets/discord.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>ClientID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>Details</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>State</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>LargeImageKey</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>SmallImageKey</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>ButtonName1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>ButtonName2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>LargeImageText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>SmallImageText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>ButtonUrl1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>ButtonUrl2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>Timestamp</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
